study.md

```diff
@@ -54,5 +57,6 @@ like enumerables?  How are they different?  Please compare and contrast these
 types.
 
 ```md
-<!-- your answer here -->
+Stream has sequence of data - items which are processed one at a time more often then in large batch.
```

Comment:
Streams should generally be treated as infinite, even though they can be finite, and enumerables should generally be treated as finite, even though they can be infinite.